boefjes/boefjes/plugins/kat_wpscan/normalize.py

```diff
@@ -8,7 +8,7 @@
 from octopoes.models.ooi.findings import CVEFindingType, Finding
 
 
-def run(normalizer_meta: NormalizerMeta, raw: Union[bytes, str]) -> Iterator[OOI]:
+def run(normalizer_meta: NormalizerMeta, raw: Union[bytes, str]) -> Iterable[OOI]:
```

Comment:
Iterable is not defined, as you have forgotten to import it there. Iterator is till imported instead.